pandas/tests/extension/base/methods.py

```diff
@@ -44,6 +45,15 @@ def test_argsort_missing(self, data_missing_for_sorting):
         expected = pd.Series(np.array([1, -1, 0], dtype=np.int64))
         self.assert_series_equal(result, expected)
 
+    @pytest.mark.parametrize('na_position, expected', [
+        ('last', np.array([2, 0, 1])),
+        ('first', np.array([1, 2, 0]))
+    ])
+    def test_nargsort(self, data_missing_for_sorting, na_position, expected):
+        # GH 25439
+        result = nargsort(data_missing_for_sorting, na_position=na_position)
+        tm.assert_numpy_array_equal(result, expected, check_dtype=False)
```

Comment:
Why is `check_dtype` False? Was it an Appveyor failure?

If you change the expected to `np.array([2, 0, 1], dtype='int64')` I think you may be OK, and can remove `check_dtype=False`.